### PR TITLE
Interim docs for Spring Session for GF

### PR DESCRIPTION
**Pull request by jomartin-999**

### CODE REVIEW

File: documentation/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+### 1.0.0
+* Initial release of the Spring Session For VMware GemFire dependencies for **GemFire 10**. 
+
+
+## Spring Session 2.6, 2.7, 3.0 & GemFire 9.15
+
+### 1.1.1
+
+* Update Spring Session 2.6 to Spring Session 2.6.14
```
Comment (kohlmu-pivotal) [inline documentation/release-notes.md]: 2.6.14 -> 2.6.3

File: documentation/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* Update Spring Session 2.7 to Spring Session 2.7.10
```
Comment (kohlmu-pivotal) [inline documentation/release-notes.md]: 2.7.10 - > 2.7.1

File: documentation/release-notes.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+* Update Spring Session 3.0 to Spring Session 3.0.5
```
Comment (kohlmu-pivotal) [inline documentation/release-notes.md]: 3.0.5 -> 3.0.1

File: documentation/release-notes.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* Update all dependencies to latest patch version
+* Fixed an issue with querying a Region whose name contains a hyphen (`-`).
```
Comment (kohlmu-pivotal) [inline documentation/release-notes.md]: This is only applicable to Spring Data

File: documentation/release-notes.md (hunk 1 above)
Comment (jomartin-999) [reply documentation/release-notes.md]: Done.

File: documentation/release-notes.md (hunk 2 above)
Comment (jomartin-999) [reply documentation/release-notes.md]: Done

File: documentation/release-notes.md (hunk 4 above)
Comment (jomartin-999) [reply documentation/release-notes.md]: Done

**Review by kohlmu-pivotal — APPROVED** [review]

**Pull request merged**